package.json

```diff
@@ -103,14 +103,17 @@
   "homepage": "https://github.com/pmndrs/valtio",
   "packageManager": "pnpm@8.15.0",
   "dependencies": {
-    "proxy-compare": "^3.0.0"
+    "proxy-compare": "^3.0.0",
+    "sorted-btree": "^1.8.1",
+    "valtio-dev-tools": "link:/Users/msweeney/Projects/Personal/valtio-dev-tools"
```

Comment:
This causes an error to install a built package with `npm`. Can you remove it for now?

```bash
❯ npm i https://pkg.pr.new/pmndrs/valtio/valtio@d720a96
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:/Users/msweeney/Projects/Personal/valtio-dev-tools
```

Reply:
oh my bad - that's a different lib I'm working on from a different branch. Yeah I'll remove that.